packages/peregrine/lib/List/item.js

```diff
@@ -54,7 +54,8 @@ Item.propTypes = {
     item: PropTypes.any.isRequired,
     itemIndex: PropTypes.number.isRequired,
     render: PropTypes.oneOfType([PropTypes.func, PropTypes.string]).isRequired,
-    uniqueID: PropTypes.number
+    uniqueID: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
```

Comment:
Are the uses of `uniqueID` within this component capable of handling either?

Reply:
I think this uses a set downstream which means we need to make sure to use a single type.

Reply:
Yes, we just use it as key for a component. Since react does not allow users to send a prop called key to its children, I am just passing it down as uniqueID. Once it reaches the `Item` component, it is used as `key` which can be `number` or `string`.